crates/core/src/host/scheduler.rs

```diff
@@ -87,8 +87,7 @@ impl SchedulerStarter {
     // time to make it better right now.
     pub fn start(mut self, module_host: &ModuleHost) -> anyhow::Result<()> {
         let mut queue: DelayQueue<QueueItem> = DelayQueue::new();
-        let ctx = &ExecutionContext::internal(self.db.address());
-        let tx = self.db.begin_tx();
+        let tx = self.db.begin_tx(Workload::Internal);
```

Comment:
Is this init?

Reply:
I don't follow. What you expect here?

Reply:
I just want to make sure `Workload::Internal` is the correct workload type. Is this called during init?